_overviews/contributors/index.md

```diff
@@ -46,62 +46,45 @@ use the correct sbt version.
-repository with the following content:
+Our example uses Github Actions. Actions are enabled by default on most repositories. You can verify if that is
+the case and enable them in the *Actions* section of the *Settings* tab of the repository.
 
-~~~ yaml
-language: scala
-~~~
-
-Push your changes and check that Travis CI triggers a build for your repository.
-
-Travis CI tries to guess which build tool your project uses and executes a default command to run the project tests.
-For instance, if your repository contains a `build.sbt` file in the root directory, Travis CI executes the
-`sbt ++$TRAVIS_SCALA_VERSION test` command, where the `TRAVIS_SCALA_VERSION` variable is, by default, set to an
-arbitrary Scala version (`2.12.8`, at the time these lines are written), which could be inconsistent with the
-`scalaVersion` defined in your `build.sbt` file.
-
-To avoid this potential inconsistency, you want to use one Scala version definition as a single source of truth.
-For instance, the [sbt-travisci](https://github.com/dwijnand/sbt-travisci) plugin lets you define the Scala version
-in the `.travis.yml` file, and then forwards this version to your sbt build definition. Alternatively, you can
-override the default command run by Travis CI to use the Scala version defined by the `scalaVersion` settings of
-your build.
+With Actions enabled, you can create a *workflow definition file*. A **workflow** is an automated procedure,
+composed of one or more jobs. A **job** is a set of sequential steps that are executed on the same runner.
+A **step** is an individual task that can run commands; a step can be either an *action* or a shell command.
+An **action** is the smallest building block of a workflow, it is possible to reuse community actions or to
+define new ones.
 
-The latter approach is the one used in this guide. Override the command run by Travis CI by adding the folliwng
-lines to your `.travis.yml` file:
+To create a workflow, create a *yaml* file in the `.github/workflows` folder in the repo, for example
+`.github/workflows/ci.yml` with the following content:
 
 ~~~ yaml
+name: Continuous integration
+on: [push]
+
 jobs:
-  include:
-    - stage: test
-      script: sbt test
+  Tests: 
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2 # Retrieve the content of the repository
+      - uses: olafurpg/setup-scala@v10 # Set up a jdk and sbt
+      - name: unit tests # Custom action consisting of a shell command
+        run: sbt test
 ~~~
 
-Travis CI will now execute the `sbt test` command, which uses the Scala version from the build definition.
+This workflow is called *Continuous integration* and will run every time that one or more commits are pushed to
+the repository. It contains only one job called *Tests*, which will run on an Ubuntu runner and that is composed
+of three actions. `setup-scala` uses the sbt version specified in `project/build.properties` and the Scala
+version defined in your `build.sbt`.
```

Comment:
I don’t think this is correct. I think setup-scala installs an sbt launcher. When the launcher is executed (with `sbt test`), then the appropriate version of sbt and Scala compiler are downloaded.

Reply:
I think that you are right. [This line](https://github.com/olafurpg/setup-scala/blob/05b368ddd620abc33fd3412cb2b33b54868a78b5/src/install.ts#L105) action installs dwijnand/sbt-extras. 

However I think that my statement is not necessarily wrong. Even if the version of sbt downloaded is not the one in the `build.properties`, the one used by all subsequent invocation will be correct, assuming that the users performs the checkout action.

How would you rephrase it?

Reply:
What about the following?

> The action `setup-scala` installs the sbt launcher in the runner, then the job runs `sbt +test`, which loads the sbt version specified in `project/build.properties`, and runs the project tests using the Scala version defined in the file `build.sbt`.

Reply:
Thanks